Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasket-AddNonprofitsViewController.m

```diff
@@ -71,9 +72,19 @@ - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(N
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
     Nonprofit *nonprofit = self.allNonprofits[indexPath.row];
     //TURN COLOR CHANGE OR SELECTION UI
+    NonprofitListCell *cell = [tableView cellForRowAtIndexPath:indexPath];
+    cell.backgroundColor = [UIColor blueColor];
```

Comment:
You should use the `selectionStyle` property from `UITableViewCell`, which can be set from the interface builder too. 
![Screen Shot 2020-07-21 at 5 17 39 PM](https://user-images.githubusercontent.com/1676646/88119762-38569800-cb76-11ea-9a50-d291a6ea460b.png)